src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.tsx

```diff
@@ -17,10 +19,12 @@ import styles from './GeneView.scss';
 
 type GeneViewProps = {
   geneId: string | null;
+  selectedGeneTabName: string | null;
```

Comment:
Does the selector ever return null?

Also, a string or an enum?

Reply:
Changed it to enum